testing/netatalk/netatalk.initd

```diff
@@ -0,0 +1,21 @@
+#!/sbin/openrc-run
+
+name=netatalk
+command="/usr/sbin/$name"
+command_args="-F $netatalk_config"
+command_background="yes"
+
+extra_started_commands="reload"
+
+pidfile="/run/$name.pid"
+
+depend() {
+	need net
+	after firewall
+}
+
+stop() {
+	ebegin "Stopping $name"
+	start-stop-daemon -K -p /var/lock/netatalk
```

Comment:
The built in stop command didn't work. I'm not familiar with openrc-stuff so I just made-it-work(tm).